README.rst

```diff
@@ -27,22 +27,11 @@ For a general overview and example of electrical model building, please refer to
 
 Note that this package only covers e-model building based on patch-clamp data and that it relies solely on the `NEURON <https://www.neuron.yale.edu/neuron/>`_ simulator.
 
-Citation
---------
-
-When you use the BluePyEModel software or method for your research, we ask you to cite the following publication (this includes poster presentations):
-
-.. code-block::
-
-    @software{bluepyemodel_zenodo,
-      author       = {Damart, Tanguy and Jaquier, Aurélien and Arnaudon, Alexis and Mandge, Darshan and Van Geit, Werner and Kilic, Ilkan},
-      title        = {BluePyEModel},
-      month        = aug,
-      year         = 2023,
-      publisher    = {Zenodo},
-      doi          = {8283490},
-      url          = {https://doi.org/10.5281/zenodo.8283490}
-    }
+How to cite
+===========
```

Comment:
Maybe you could keep the simple dash for title here, no?

Reply:
Thanks, fixed in the last commit